### PR TITLE
lint code in 'server' folder

### PR DESCRIPTION
**Pull request by vladima**

### CODE REVIEW

File: Jakefile.js [hunk 1]
```diff
@@ -898,7 +900,9 @@ function lintFileAsync(options, path, cb) {
     });
 }
 
-var lintTargets = compilerSources.concat(harnessCoreSources);
+var lintTargets = compilerSources
+    .concat(harnessCoreSources)
+    .concat(serverOnlySources.filter(function(p) { return path.basename(p) !== "node.d.ts"; }));
```
Comment (mhegazy) [inline Jakefile.js]: this is actually our fork of node.d.ts, we should lint it too.

**Comment by mhegazy** [comment]
:+1:

File: Jakefile.js (hunk 1 above)
Comment (vladima) [inline Jakefile.js]: ok, I was not sure about that

**Comment by mhegazy** [comment]
thanks!

**Pull request merged**